contracts/savings/SavingsManager.sol

```diff
@@ -231,12 +337,12 @@ contract SavingsManager is ISavingsManager, PausableModule {
 
             // 4. Distribute the interest
             //    Calculate the share for savers (95e16 or 95%)
-            uint256 saversShare = interestCollected.mulTruncate(savingsRate);
+            uint256 saversShare = interestCollected.add(newReward).mulTruncate(savingsRate);
```

Comment:
Does this fix a minor issue due to previously not truncating the new reward?

Reply:
Yes, "streamed" rewards (via liquidator) were not being taken into account for transferring to governance.